frontend/src/components/Nav.tsx

```diff
@@ -96,7 +98,9 @@ const Nav : FunctionComponent = () : ReactElement => {
     return (
       <>
         <Link href="/assessments">assessments</Link>
-        <Link href="/interventions">interventions</Link>
+        <Link href="/interventions" onClick={() => firebase.auth().signOut()}>
```

Comment:
Can we create an action for this? Cuz i assume when you sign out you also want to change the reducer sign_in to be false right?

Reply:
At some point yes, but I haven't actually thought enough about how sign out will actually work. I think that will end up being its own story, so I will punt on this for now if that is ok.

Reply:
okie